src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -468,6 +468,8 @@ public function handleError(int $type, string $message, string $file, int $line)
                 return true;
             }
         } else {
+            $this->handleTriggerDeprecationFunction(debug_backtrace(0), $type, $file, $line);
```

Comment:
Should we reuse this backtrace instead of calling $exception->getTrace() later? I don't know when exception backtraces are "generated".